src/main/java/org/spongepowered/common/SpongeImplHooks.java

```diff
@@ -334,4 +345,31 @@ public static void onCraftingRecipeRegister(CraftingRecipe recipe) {
         }
         return Optional.of(((CraftingRecipe) recipe));
     }
+
+    public static Text getAdditionalCommandDescriptions() {
+        return Text.EMPTY;
+    }
+
+    public static void registerAdditionalCommands(ChildCommandElementExecutor flagChildren, ChildCommandElementExecutor nonFlagChildren) {
+        // Overwritten in SpongeForge
+    }
+
+    public static List<PluginContainer> getSortedPluginList() {
```

Comment:
To clarify actually. It is overwritten by SpongeForge to remove any mods from the list.